demo/qat-init.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+ENABLED_QAT_PF_PCIIDS=${ENABLED_QAT_PF_PCIIDS:-37c8 4940}
```

Comment:
```suggestion
ENABLED_QAT_PF_PCIIDS=${ENABLED_QAT_PF_PCIIDS:-0434 0435 19a3 37c8 6f54 4940 18a0}
```

Reply:
Ok.! What do you think about this then?
https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/deployments/operator/samples/deviceplugin_v1_qatdeviceplugin.yaml#L14

Well, I know they do not need to be same,, but I do not see any reason either why they have to be different.!

Reply:
fair point but it changes the existing functionality without a message anywhere so I thought you just missed my earlier comment

Reply:
@mythi Did you "resolve"? Idk why it was changed to "resolve"..!
I did not get your point yet..!